extensions/postgres/schema/xdb_sys_schema.sql

```diff
@@ -15,6 +15,7 @@ CREATE TABLE xdb_sys_process_executions(
     timeout_seconds INTEGER,
     history_event_id_sequence INTEGER,
     state_execution_sequence_maps jsonb NOT NULL , -- some maps from stateId and sequence number
+    state_execution_waiting_queues jsonb, -- { queue_1: [(state_id, state_id_sequence, needed_count), ...], }
```

Comment:
This field will be used to expedite the process of locating the target state for message consumption.